docs/scripts/buildApi.ts

```diff
@@ -1372,7 +1378,7 @@ async function run(argv: {
         workspaceRoot,
       });
     } catch (error) {
-      error.message = `${component.filename}: ${error.message}`;
+      error.message = `${path.relative(process.cwd(), component.filename)}: ${error.message}`;
```

Comment:
I don't think the full path is relevant. Error message is now smaller. 

The displayed path is still interpreted by VSCode as a path and therefore handled as a link.